stdlib/typing.pyi

```diff
@@ -173,14 +173,14 @@ def overload(func: _F) -> _F: ...
 # Otherwise, flake8 erroneously flags them as undefined.
 # `_SpecialForm` objects in typing.py that are not used elswhere in the same file
 # do not need the default value assignment.
```

Comment:
Need to delete the multiline comment immediately above this line explaining the `= ...`s if we're getting rid of the `= ...`s